x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/endpoint_details_tabs.tsx

```diff
@@ -6,16 +6,17 @@
  */
 
 import { useDispatch } from 'react-redux';
-import React, { memo, useCallback, useMemo, useState } from 'react';
+import React, { memo, useCallback, useMemo } from 'react';
 import { EuiTab, EuiTabs, EuiFlyoutBody, EuiTabbedContentTab, EuiSpacer } from '@elastic/eui';
 import { EndpointIndexUIQueryParams } from '../../../types';
 import { EndpointAction } from '../../../store/action';
 import { useEndpointSelector } from '../../hooks';
 import { getActivityLogDataPaging } from '../../../store/selectors';
 import { EndpointDetailsFlyoutHeader } from './flyout_header';
+import { useNavigateByRouterEventHandler } from '../../../../../../common/hooks/endpoint/use_navigate_by_router_event_handler';
 
 export enum EndpointDetailsTabsTypes {
-  overview = 'overview',
+  overview = 'details',
```

Comment:
QQ: This will change the url params behaviour and all old urls will not work anymore (urls with show=overview). Is this expected? My concern is about users storing some urls somewhere (document) and then try to access it directly by copy paste it in the browser. With this change those url's wouldn't work anymore.

Reply:
That's a legit concern. So we don't have a `show=overview` actually. We do have a `show=details` which is what this PR enforces when you click and select an endpoint from the endpoint list. Before this fix, we were never using `show=details` at all. 

So no, this won't break any bookmarks that people might have.

Reply:
Perfect, thanks!